src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CloseReferralController.java

```diff
@@ -0,0 +1,166 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.controllers;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.et.common.model.ccd.CCDCallbackResponse;
+import uk.gov.hmcts.et.common.model.ccd.CCDRequest;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.ethos.replacement.docmosis.helpers.ReferralHelper;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.VerifyTokenService;
+
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.CallbackRespHelper.getCallbackRespEntityNoErrors;
+
+@Slf4j
+@RequestMapping("/closeReferral")
+@RestController
+@RequiredArgsConstructor
+public class CloseReferralController {
+    private static final String INVALID_TOKEN = "Invalid Token {}";
+    private final VerifyTokenService verifyTokenService;
+    private final ReferralHelper referralHelper;
+    private static final String COMPLETE_REFERRAL_BODY = "<hr>"

```

Comment:
Is this the correct name for it? we have the same name in other controllers that have different texts. Perhaps `CLOSE_REFERRAL_BODY` would be better?